jetpackperformancescanner/src/commonMain/kotlin/com/qamar/composescanner/TrackerModifier.kt

```diff
@@ -12,15 +14,22 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.drawWithContent
 import androidx.compose.ui.geometry.Offset
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
 import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.text.drawText
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.rememberTextMeasurer
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import com.qamar.composescanner.Border
+import com.qamar.composescanner.RecompositionTrackerTheme
 
 @Composable
-fun Modifier.trackRecompositions(): Modifier {
+fun Modifier.trackRecompositions(
+    border: Border = RecompositionTrackerTheme.border,
+    textStyle: RecompositionTextStyle = RecompositionTrackerTheme.textStyle,
+    contentPadding: PaddingValues = RecompositionTrackerTheme.contentPadding
```

Comment:
I would still prefer to pass one object for that. Like 
```suggestion
RecompostionTrackerProperties(
 RecompositionTrackerTheme.border,
RecompositionTrackerTheme.textStyle,
RecompositionTrackerTheme.contentPadding
)
```